apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data/QueryOperator.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright 2017, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Google.Cloud.Firestore.Data
+{
+    /// <summary>
+    /// Operators for use in queries.
+    /// </summary>
+    public static class QueryOperator
```

Comment:
Why not use an enum instead of strings?

Reply:
I originally had it as an enum, but went to strings so that they *could* be written as just "==" etc if necessary. I think it was probably a bad idea - will go back to enums.